app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less

```diff
@@ -123,7 +123,7 @@
                 .reviews-actions {
                     font-size: @font-size__s;
                     margin-top: 5px;
-                    text-transform: lowercase;
+                    text-transform: none;
```

Comment:
How about just removing text-transform? :)

Reply:
Yes we can remove text-transform too

Reply:
@surbhi-ranosys can you do that then? ;)

Reply:
@Karlasa Yes sure

Reply:
@Karlasa I have removed text transform can you please check